File: Java/Carro.java
package Java;

public class Carro {

    Integer id;
    String licencia;
    Persona conductor;
    Integer pasajeros;

    public Carro(String licencia, Persona conductor) {
        this.licencia = licencia;
        this.conductor = conductor;
    }

    void ImprimirInfo() {
        System.out.println(id + " " + licencia + " " + conductor + " " + pasajeros);
    }
}

File: PHP/carro.php

<?php

require_once "persona.php"; 

class Carro{
    
    public $id;
    public $licencia;
    public $conductor;
    public $pasajeros;

    public function __construct($licencia, $conductor)
    {
        $this-> licencia = $licencia;
        $this -> conductor = $conductor;
    }

    public function ImprimirInfo() {
        echo " Licencia: $this->licencia, Conductor: {$this->conductor->nombre}, Documento: {$this->conductor->documento}";

    }
}
